sdk/core/azure-core/src/main/java/com/azure/core/util/BinaryData.java

```diff
@@ -1459,4 +1459,12 @@ public Flux<ByteBuffer> toFluxByteBuffer() {
     public Long getLength() {
         return content.getLength();
     }
+
+    /**
+     * TODO (kasobol-msft) add docs.
+     * @return A flag.
+     */
+    public boolean isRepeatable() {
```

Comment:
Should we call this method `isReplayable`?

Reply:
will do in final PR.